src/components/FieldInput.js

```diff
@@ -17,18 +17,22 @@ const useStyles = makeStyles({
 });
 
 export default function FieldInput(props) {
-  const { label, onChange, placeholder, variant } = props;
+  const { label, onChange, placeholder, variant, error = null } = props;
   const classes = useStyles();
 
   return (
     <div className={classes.root}>
       {label ? <h3 className={classes.label}>{label}</h3> : null}
+
       <TextField
         fullWidth
         margin="normal"
         variant={variant || 'standard'}
         placeholder={placeholder}
         onChange={onChange}
+        {...(error && { error: true, helperText: error })}
+        helperText={error}
+        autoComplete
```

Comment:
sorry i was wrong 
autoComplete doesn't work right now but based on the [API ](https://material-ui.com/api/text-field/) you need to pass in a string of the right fiekd type (ie address, phone, etc). Refer to the docs.